functional-tests/factory/factory.py

```diff
@@ -289,7 +289,8 @@ def create_prover_client(
             "--bitcoind-password", bitcoind_config["bitcoind_pass"],
             "--datadir", datadir,
             "--native-workers", str(settings.native_workers),
-            "--loop-interval", str(settings.loop_interval)
+            "--loop-interval", str(settings.loop_interval),
```

Comment:
I didn't notice this when it was originally added but this is a *very* vague arg name, even if it's documented.  Should be named something like `--poll-interval` or something.